types/mempool/priority_nonce.go

```diff
@@ -382,7 +395,7 @@ func (mp *PriorityNonceMempool[C]) reorderPriorityTies() {
 		if mp.priorityCounts[key.priority] > 1 {
 			newKey := key
 			newKey.weight = senderWeight(mp.cfg.TxPriority, key.senderElement)
-			reordering = append(reordering, reorderKey[C]{deleteKey: key, insertKey: newKey, tx: node.Value.(sdk.Tx)})
+			reordering = append(reordering, reorderKey[C]{deleteKey: key, insertKey: newKey, tx: node.Value.(MempoolTx).Tx})
```

Comment:
seems it's not a mempool.MempoolTx

Reply:
yes, priority index set `sdk.Tx`, sender index set `MempoolTx`